src/migrations/migrations.ts

```diff
@@ -80,12 +80,15 @@ export const migrations = {
         thighLeft: true,
         thighRight: true,
       },
+      percentage: {
+        bodyfat: true,
+      },
```

Comment:
Migrations are to ensure we modify existing users storages, so they'd conform to the types/schema defined in types.ts, and avoid errors when e.g. you they try to access `storage.settings.stats.percentage[0].value`, and it throws because their storages don't have `storage.settings.stats.percentage`.

To make it work, you need to create a new entry at the end of this dict. Modifying old entry won't work for existing accounts, since they already have this migration entry applied.

You need to add a new key like "yyyymmddhhmmss_add_percentage_to_stats" or something like that, where "yyyymmddhhmmss" is current date, something like "20230618121655", add it to the end of this dict, and modify settings to `statsEnabled.percentage ||= {bodyfat: false}` (so users wouldn't be surprised why they got bodyfat setting suddently enabled :)), and also `storage.stats.percentage = storage.stats.percentage || {}`.